### PR TITLE
[Improvement] [clickhouse-e2e] Enhance ClickHouse E2E testing to trigger multiple checkpoints

### PR DESCRIPTION
**Pull request by Carl-Zhou-CN**
Enhance ClickHouse E2E testing to trigger multiple checkpoints

#5468 

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java [hunk 1]
```diff
@@ -76,7 +76,7 @@
 
 public class ClickhouseIT extends TestSuiteBase implements TestResource {
     private static final Logger LOG = LoggerFactory.getLogger(ClickhouseIT.class);
-    private static final String CLICKHOUSE_DOCKER_IMAGE = "yandex/clickhouse-server:22.1.3.7";
+    private static final String CLICKHOUSE_DOCKER_IMAGE = "clickhouse/clickhouse-server:latest";
```
Comment (ruanwenjun) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java]: It's not a good idea to change the image to the latest version.

Comment (Carl-Zhou-CN) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java]: @ruanwenjun  ClickhouseSinkCDCChangelogIT uses clickhouse/clickhouse-server:latest. Should we unify it to yandex/clickhouse-server:22.1.3.7

Comment (Hisoka-X) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java]: I think @ruanwenjun means don't use `latest`, the version it represents can be changed at any time. Just use a certain version.

> ClickhouseSinkCDCChangelogIT uses clickhouse/clickhouse-server:latest. Should we unify it to yandex/clickhouse-server:22.1.3.7

Just unify use version which `clickhouse/clickhouse-server:latest` represents. (Seem like is `23.3.13.6`)

Comment (Carl-Zhou-CN) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java]: > Just unify use version which `clickhouse/clickhouse-server:latest` represents. (Seem like is `23.3.13.6`)

ok，I'll replace it with that version
image

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**